pkg/storaged/lvol-tabs.jsx

```diff
@@ -250,19 +267,84 @@
         return;
     }
 
+    const pvs_as_spaces = pvs_to_spaces(client, client.vgroups_pvols[vgroup.path].filter(pvol => pvol.FreeSize > 0));
+
+    let initial_pvs;
+
+    if (!has_subvols)
+        initial_pvs = pvs_as_spaces;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230912-121317-ed28e52d-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)